files/Aboot/boot0.j2

```diff
@@ -130,7 +130,7 @@ platform_specific() {
         aboot_machine=arista_7260cx3_64
     fi
     if [ "$platform" = "rook" ]; then
-        varlog_size=200
+        varlog_size=4096
```

Comment:
I thought we are going to wait for Arista to get back on flash disk size and based on that, implement a logic to determine varlog_size for all platforms

Reply:
Sorry Prince,

I didn't see this comment before merging.

You are absolutely right that we are waiting for a better solution. I need more /var/log space since I am experiencing some issues that I need more logs to debug. This is a gap filling  mechanism.

Cheers,
Ying